src/extension/background-script/actions/ln/request.ts

```diff
@@ -43,13 +43,15 @@ const request = async (
       throw new Error("Could not find an allowance for this host");
     }
 
+    const connectorName = connector.constructor.name.toLowerCase();
     // prefix method with webln to prevent potential naming conflicts (e.g. with nostr calls that also use the permissions)
-    const weblnMethod = `${WEBLN_PREFIX}${methodInLowerCase}`;
+    const legacyWeblnMethod = `${WEBLN_PREFIX}${methodInLowerCase}`;
```

Comment:
maybe we just drop the legacy method. 
Users will have to approve the permission again which I think is OK. and the code is easier.